.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,5 +5,35 @@ Thank you for your Pull Request!
 
 If you are new to contributing to Mono, please try to do your best at conforming to our coding guidelines http://www.mono-project.com/community/contributing/coding-guidelines/ but don't worry if you get something wrong. One of the project members will help you to get things landed.
 
-Does your pull request fix any of the existing issues? Please use the following format: Fixes #issue-number
+Here are a few things to think about (see below for more details). Please check each option after the PR is created.
 -->
+
+- Should this pull request have release notes?
+  - [ ] Yes
+  - [ ] No
+- Do these changes need to be back ported?
+  - [ ] Yes
+  - [ ] No
+
+Reviewers: please consider these questions as well! :heart:
+
+<!-- Use this section if the pull request has release notes.
+**Release notes**
+
+Fixed case XXXXXX @username:
```

Comment:
Should this be github or Unity username?

Reply:
I checked in a Ono PR using a GitHub username, and it did not error. Github username is OK.